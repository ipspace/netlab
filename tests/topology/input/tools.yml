nodes:
- name: a_eos
  device: eos

tools:
  graphite:
  suzieq:

