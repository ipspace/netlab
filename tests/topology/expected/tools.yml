input:
- topology/input/tools.yml
- package:topology-defaults.yml
name: input
nodes:
  a_eos:
    af:
      ipv4: true
    box: arista/veos
    device: eos
    id: 1
    interfaces: []
    loopback:
      ifindex: 0
      ifname: Loopback0
      ipv4: 10.0.0.1/32
      neighbors: []
      type: loopback
      virtual_interface: true
    mgmt:
      ifname: Management1
      ipv4: 192.168.121.101
      mac: 08:4f:a9:00:00:01
    name: a_eos
provider: libvirt
tools:
  graphite:
    runtime: docker
  suzieq:
    runtime: docker
